src/bench/bench_bitcoin.cpp

```diff
@@ -18,6 +18,8 @@
 
 static const char* DEFAULT_BENCH_FILTER = ".*";
 static constexpr int64_t DEFAULT_MIN_TIME_MS{10};
+/** Priority level default value, run "all" priority levels */
+static const std::string DEFAULT_PRIORITY_LEVEL{"all"};
```

Comment:
Not a huge concern, but I'm not terribly excited about this string reference. Would be nicer imo to have the default be a `uint8_t` so it can't go out of sync, but that would require the `PriorityToString` fn again which may be overkill.

Reply:
yeah. Same feeling here. But.. would say to leave it as is for now, otherwise it would mean to re-introduce the `PriorityToString` function only to stringify this priority level.